pages/windows/shutdown.md

```diff
@@ -0,0 +1,35 @@
+# shutdown
+
+> A tool for shutting down, restarting or logging off a machine.
+
+- Shutdown the current machine:
+
+`shutdown /s`
+
+- Restart the current machine:
+
+`shutdown /r`
+
+- Hibernate the current machine:
+
+`shutdown /h`
+
+- Log off the current machine:
+
+`shutdown /l`
+
+- Specify a timeout in seconds to wait before shutting down:
+
+`shutdown /s /t {{seconds}}`
+
+- Specify a comment for the shutdown reason:
+
+`shutdown /s /c "{{reason}}"`
+
+- Abort a shutdown sequence:
+
+`shutdown /a`
+
+- Shutdown a remote machine:
+
+`shutdown /m {{\\computer}}`
```

Comment:
This seems unclear. Should we mention specify if its the computer name or hostname/IP?

Reply:
I guess I can add `using it's hostname or computer name:` or something along those lines.

Reply:
Maybe we could update the example instead to `\\hostname`?

Reply:
Yes, `\\hostname` is definitely clearer